requirements.txt

```diff
@@ -58,7 +58,6 @@ requests==2.32.1
 rich~=13.4.2
 s3transfer~=0.6.1
 six~=1.16.0
-sphinx==7.1.2
```

Comment:
This is already included in the dev-requirements file.

Reply:
I'm not sure where I can access the sphinx documentation. If anyone knows where I can, let me know and I'll make sure everything is 💯 before we push.

Reply:
I think it should be in the `docs/_build/html` directory

Reply:
I think another task would be to add validation that the sphinx files are in place. I may do that later.